index.html

```diff
@@ -33,19 +37,95 @@ <h1 class="logo">
         hesitate and contact me.
       </p>
 
-      <button type="button">Lets Connect</button>
+      <button type="button" class="connect">Lets Connect</button>
       <img class="design-one" src="bgbg.png" alt="hardwork bro" />
     </section>
 
     <section class="Projects">
       <div class="headingtwo">Projects</div>
-      <img src="Rectangle21.png" alt="project 1" />
+
       <div class="project-info">
-        <div>Project name goes here</div>
-        <ul>
-          <li>HTML/CSS</li>
-          <li>Ruby on Rails</li>
-          <li>JavaScript</li>
+        <img class="thumbnails" src="Rectangle21.png" alt="project 1" />
+        <div class="project-name">AI blog writer</div>
```

Comment:
- [ ] Kindy change every `div class="project-name"` to one of the `hx(h2-h6)` tags. Using `hx` tags for name titles or subtitles makes your HTML code semantic. 👍